autogpts/autogpt/autogpt/url_utils/validators.py

```diff
@@ -2,10 +2,11 @@
 import re
 from typing import Any, Callable, ParamSpec, TypeVar
 from urllib.parse import urljoin, urlparse
+from autogpt.config import Config
```

Comment:
this is gonna throw a linting error

Reply:
Added flake8 to VS code, sorry there was some issue with flake earlier.